packages/nodebrew.rb

```diff
@@ -3,45 +3,62 @@
-    system "ln -s #{CREW_PREFIX}/share/nodebrew/current/bin/npm #{CREW_DEST_PREFIX}/bin/"
-    system "ln -s #{CREW_PREFIX}/share/nodebrew/current/bin/npx #{CREW_DEST_PREFIX}/bin/"
-    system "ln -s #{CREW_PREFIX}/share/nodebrew #{CREW_DEST_DIR}/$HOME/.nodebrew"
-    system "ln -sf #{CREW_PREFIX}/share/nodebrew $HOME/.nodebrew"
+    FileUtils.mkdir_p "#{CREW_DEST_DIR}#{@home}"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
+    FileUtils.ln_s "#{CREW_PREFIX}/share/nodebrew/nodebrew", "#{CREW_DEST_PREFIX}/bin/"
+    FileUtils.ln_s "#{CREW_PREFIX}/share/nodebrew/current/bin/node", "#{CREW_DEST_PREFIX}/bin/"
+    FileUtils.ln_s "#{CREW_PREFIX}/share/nodebrew/current/bin/npm", "#{CREW_DEST_PREFIX}/bin/"
+    FileUtils.ln_s "#{CREW_PREFIX}/share/nodebrew/current/bin/npx", "#{CREW_DEST_PREFIX}/bin/"
+    FileUtils.ln_s "#{CREW_PREFIX}/share/nodebrew/current/bin/node", "#{CREW_DEST_PREFIX}/bin/nodejs"
+    FileUtils.ln_s "#{CREW_PREFIX}/share/nodebrew", "#{CREW_DEST_DIR}#{@home}/.nodebrew"
+    FileUtils.ln_sf "#{CREW_PREFIX}/share/nodebrew", "#{@home}/.nodebrew"
   end
 
   def self.postinstall
+    FileUtils.ln_sf "#{CREW_PREFIX}/share/nodebrew/default", "#{CREW_PREFIX}/share/nodebrew/current"
     puts
-    puts "Nodebrew completion support is available for the following shells:".lightblue
-    puts "bash fish zsh".lightblue
+    puts 'Nodebrew completion support is available for the following shells:'.lightblue
+    puts 'bash fish zsh'.lightblue
     puts
-    puts "To add nodebrew completion for bash, execute the following:".lightblue
+    puts 'To add nodebrew completion for bash, execute the following:'.lightblue
     puts "echo '# nodebrew completion' >> ~/.bashrc".lightblue
     puts "echo 'if [ -f #{CREW_PREFIX}/share/nodebrew/completions/bash/nodebrew-completion ]; then' >> ~/.bashrc".lightblue
     puts "echo '  source #{CREW_PREFIX}/share/nodebrew/completions/bash/nodebrew-completion' >> ~/.bashrc".lightblue
     puts "echo 'fi' >> ~/.bashrc".lightblue
-    puts "source ~/.bashrc".lightblue
+    puts 'source ~/.bashrc'.lightblue
     puts
-    puts "To complete the installation, execute the following:".lightblue
+    puts 'To complete the installation, execute the following:'.lightblue
     puts "echo 'export PATH=\$HOME/.nodebrew/current/bin:\$PATH' >> ~/.bashrc && source ~/.bashrc".lightblue
     puts
-    puts "To install the latest node, execute:".lightblue
-    puts "nodebrew install-binary latest".lightblue
-    puts "nodebrew use latest".lightblue
+    puts 'To install the latest node, execute:'.lightblue
+    puts 'nodebrew install-binary latest'.lightblue
+    puts 'nodebrew use latest'.lightblue
     puts
-    puts "To uninstall, execute the following:".lightblue
-    puts "crew remove nodebrew".lightblue
+    puts 'To uninstall, execute the following:'.lightblue
+    puts 'crew remove nodebrew'.lightblue
     puts "rm -rf #{CREW_PREFIX}/share/nodebrew".lightblue
```

Comment:
Add in `def self.remove` section instead?

Reply:
Is `def self.remove` automatically run? Because this should be optional, no?

Reply:
Does this look right?